Dockerfile

```diff
@@ -251,6 +232,9 @@ RUN set -x \
 	&& cp bin/ctr /usr/local/bin/docker-containerd-ctr \
 	&& rm -rf "$GOPATH"
 
+# Used to detect running from within a container
+ENV FROM_DOCKERFILE=1
```

Comment:
Is is valid to test the existence of `/.dockerenv` instead of introducing this environment variable?

I'm not sure `/.dockerenv`  will continue to exist in future releases, but many people have been already using that file for checking whether running in Docker:
https://github.com/search?utf8=%E2%9C%93&q=if+-f+%2F.dockerenv&type=Code&ref=searchresults